terragrunt/aws/alb/variables.tf

```
@@ -0,0 +1,86 @@
+# Required variables
+variable "hosted_zone_id" {
+  description = "The hosted zone ID to create DNS records in"
+  type        = string
+}
+
+variable "hosted_zone_name" {
+  description = "Route53 hosted zone ID that will hold our DNS records"
+  type        = string
+}
+
+variable "vpc_id" {
+  description = "The VPC id of the url shortener"
+  type        = string
+}
+
+variable "vpc_cidr_block" {
+  description = "IP CIDR block of the VPC"
+  type        = string
+}
+
+variable "vpc_public_subnet_ids" {
+  description = "Public subnet ids of the VPC"
+  type        = list(string)
+}
+
+variable "app_port" {
+  description = "The port the application runs on"
+  type        = number
+  default     = 8000
+}
+
+variable "ssl_policy" {
+  description = "The SSL policy for the HTTPS listener"
+  type        = string
+  default     = "ELBSecurityPolicy-TLS-1-2-2017-01"
```

Could you bump this to the following policy (assuming Blawx supports only TLS 1.3+)

```suggestion
  default     = "ELBSecurityPolicy-TLS13-1-3-FIPS-2023-04"
```